$MirOS: ports/sysutils/pxegrub/patches/patch-stage2_asm_S,v 1.1.7.1 2005/03/18 15:52:19 tg Exp $
--- stage2/asm.S.orig	Sat Jun 19 16:55:00 2004
+++ stage2/asm.S	Mon Feb 14 21:14:14 2005
@@ -98,7 +98,7 @@ VARIABLE(version_string)
 	.string VERSION
 VARIABLE(config_file)
 #ifndef STAGE1_5
-	.string "/boot/grub/menu.lst"
+	.string "/grub.cfg"
 #else   /* STAGE1_5 */
 	.long	0xffffffff
 	.string "/boot/grub/stage2"
